packages/twenty-front/src/modules/workflow/search-variables/components/SearchVariablesDropdownObjectItems.tsx

```diff
@@ -0,0 +1,74 @@
+import { DropdownMenuHeader } from '@/ui/layout/dropdown/components/DropdownMenuHeader';
+import { DropdownMenuItemsContainer } from '@/ui/layout/dropdown/components/DropdownMenuItemsContainer';
+import { DropdownMenuSearchInput } from '@/ui/layout/dropdown/components/DropdownMenuSearchInput';
+import {
+  RecordOutputSchema,
+  StepOutputSchema,
+} from '@/workflow/search-variables/types/StepOutputSchema';
+
+import { useState } from 'react';
+import {
+  IconChevronLeft,
+  MenuItemSelect,
+  OverflowingTextWithTooltip,
+  useIcons,
+} from 'twenty-ui';
+
+type SearchVariablesDropdownObjectItemsProps = {
+  step: StepOutputSchema;
+  onSelect: (value: string) => void;
+  onBack: () => void;
+};
+
+export const SearchVariablesDropdownObjectItems = ({
+  step,
+  onSelect,
+  onBack,
+}: SearchVariablesDropdownObjectItemsProps) => {
+  const [searchInputValue, setSearchInputValue] = useState('');
+  const { getIcon } = useIcons();
+  const recordOutputSchema = step.outputSchema as RecordOutputSchema;
+
+  const handleSelectObject = () => {
+    const fieldIdName = recordOutputSchema.object.fieldIdName;
+    onSelect(`{{${step.id}.${fieldIdName}}}`);
+  };
+
+  const goBack = () => {
+    onBack();
+  };
```

Comment:
we can directly use onBack and delete goBack

Reply:
updated